docs/configure_nydus.md

```diff
@@ -125,3 +125,11 @@ $ kubectl create --namespace nydus-system secret generic regcred \
 ```
 
 The Nydus snapshotter will get the new secret and parse the authorization. If your new Pod uses a private registry, then this authentication information will be used to pull the image from the private registry.
+
+## Metrics
+
+Nydusd records metrics in its own format. The metrics are exported via a HTTP server on top of unix domain socket. Nydus-snapshotter fetches the metrics and convert them in to Prometheus format which is exported via a network address. Nydus-snapshotter by default does not fetch metrics from nydusd. You can enable the nydusd metrics download by assigning an network address to `metrics.address` in nydus-snapshotter's toml [configuration file](../../misc/snapshotter/config.toml).
+
+## Diagnose
+
+By setting `enable_system_controller` to `true`,  nydus-snapshotter will start a simple HTTP serve on unix domain socket `$ROOT/system.sock` and exports some internal working status to users.
```

Comment:
Maybe we need to update it for new configuration items because another PR is modifying them.

Reply:
Seasonable, let's mark this PR as draft for now and waiting for another PR being merged